kcidb/monitor/output.py

```diff
@@ -136,9 +139,26 @@ def render(self):
             ready to send, but missing the From header.
         """
         email = EmailMessage()
-        email["Subject"] = self.message.summary + self.obj.summarize()
+        subject = self.message.summary + self.obj.summarize()
+        email["Subject"] = subject
         email["To"] = ", ".join(self.message.recipients)
         email["X-KCIDB-Notification-ID"] = self.id
         email["X-KCIDB-Notification-Message-ID"] = self.message.id
-        email.set_content(self.message.description + self.obj.describe())
+        message = self.message.description + self.obj.describe()
+        email.set_content(message)
+        escaped_subject = html.escape(subject)
+        escaped_message = html.escape(message)
+        escaped_message = tor.linkify(escaped_message)
```

Comment:
Adding Tornado as a (huge) dependency just for adding links to our notifications is not worth the trouble.
How well do you know your regular expressions? Can you use `re.sub()` to do this? We don't need anything fancy at this moment, just `http://`, `https://`, `git://` and maybe `ftp://` URLs.

Reply:
Okay

Reply:
Fixed using re.